CrewChiefV4/UserInterface/PropertiesForm.resx

```diff
@@ -120,4 +120,16 @@
   <metadata name="flowLayoutPanel1.Locked" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
     <value>True</value>
   </metadata>
+  <metadata name="searchBoxTooltip.TrayLocation" type="System.Drawing.Point, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+    <value>17, 17</value>
+  </metadata>
+  <data name="textBox1.ToolTip" xml:space="preserve">
+    <value>Hint:
+acs_ - for Assetto Corsa specific preferences
+ams_ - for Automobilista specific preferences
+r3e_ - for Raceroom specific preferences
+rf1_ - for rFactor 1 specific preferences
+rf2_ - for rFactor 2 specific preferences
+pcars_ - for Project CARS specific preferences</value>
```

Comment:
if we make the property labels all nice and friendly we'll also have to update these (as presumably we'd start the friendly name with "ACS ", "AMS ", "R3E " etc